src/App.tsx

```diff
@@ -43,11 +50,29 @@ Sentry.init({
   dsn: Config.SENTRY_DSN,
 })
 
-export const segmentClient = createClient({
+const segmentClient = createClient({
   writeKey: Config.SEGMENT_ANALYTICS_KEY,
   trackAppLifecycleEvents: true,
+  collectDeviceId: true,
+  debug: true,
```

Comment:
I guess you have to set it to false or use __DEV__

Reply:
ok